### PR TITLE
story/EUI 5454

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5454

### Change description ###
Get location details from location epimms id

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/prd/location/location.mock.ts [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
-import {ALL_COURT_LOCATIONS} from './data/location.mock.data';
-import {toEpimsLocation} from "./models/location.model";
+import { HttpMockAdapter } from '../../common/httpMockAdapter';
+import { ALL_COURT_LOCATIONS } from './data/location.mock.data';
+import { toEpimsLocation } from "./models/location.model";
 
 export const init = () => {
   const mock: MockAdapter = HttpMockAdapter.getInstance();
   const getLocationsUrl = /http:\/\/rd-professional-api-aat.service.core-compute-aat.internal\/refdata\/location\/court-venues\/venue-search\?search-string=[\w]*/;
+  const getCourtLocationsUrl = /http:\/\/rd-professional-api-aat.service.core-compute-aat.internal\/refdata\/location\/court-locations\?epimms_id=[\w]*/;
```
Comment (rayliangatos) [inline api/prd/location/location.mock.ts:10]: could this be getLocationByIdUrl

File: api/prd/location/location.mock.ts [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+
+  mock.onGet(getCourtLocationsUrl).reply(config => {
+    const requestURL = config.url;
+    const courTypeIdStartIdx = config.url.lastIndexOf('=');
+    const courTypeIdsStr = requestURL.substring(courTypeIdStartIdx + 1);
```
Comment (rayliangatos) [inline api/prd/location/location.mock.ts:15]: this should be epimmsIDIdx and epimmsID

File: api/prd/location/routes.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
 import authInterceptor from '../../lib/middleware/auth';
-import {getLocations} from './index';
+import { getCourtLocations, getLocations } from './index';
 
-export const router = express.Router({mergeParams: true});
+export const router = express.Router({ mergeParams: true });
 
 router.use(authInterceptor);
 router.get('/getLocations', getLocations);
+router.get('/getCourtLocations', getCourtLocations);
```
Comment (rayliangatos) [inline api/prd/location/routes.ts:9]: could this be getLocationById

**Review by rayliangatos — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Pull request merged**